devel/tmt/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                tmt
+version             1.34.0
+revision            0
+
+categories          devel
+supported_archs     noarch
+license             MIT
+maintainers         {sub-pop.net:link @subpop} \
+                    openmaintainer
+
+description         Test Management Tool
+long_description    The tmt tool provides a user-friendly way to work with \
+                    tests. You can comfortably create new tests, safely and \
+                    easily run tests across different environments, review \
+                    test results, debug test code and enable tests in the CI \
+                    using a consistent and concise config. \
+                    The python module and command-line tool implement the \
+                    Metadata Specification which allows storing all needed \
+                    test execution data directly within a git repository. \
+                    Together with possibility to reference remote repositories \
```

Comment:
Minor grammar problem on tis like.

Reply:
Ah. This description was copied verbatim from the project website. I suspect it was authored by a non-native English speaker. I'll fix it and see about sending a PR upstream too.